src/ansys/grantami/bomanalytics/_connection.py

```diff
@@ -382,14 +415,20 @@ def _query_arguments(
         The database key is always required. The default is only included here for convenience.
         """
 
+        config = models.CommonRequestConfig()
+        if self._max_spec_depth is not None:
+            logger.info(f"Using maximum spec to spec link depth: {self._max_spec_depth}")
```

Comment:
I know it's a mouthful, but we should probably spell it out each time

```suggestion
            logger.info(f"Using maximum specification-to-specification link depth: {self._max_spec_depth}")
```